pyproject.toml

```diff
@@ -0,0 +1,85 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "pybind11>=2", "setuptools_scm[toml]>=3.4"]
+build-backend = "setuptools.build_meta"
+
+# ==============================================================================
+
+[tool.black]
+
+    line-length = 118
+    target-version = ['py36','py37','py38']
+    skip-string-normalization = true
+
+
+[tool.check-manifest]
+    ignore = [
+    'PKG-INFO',
+    '*.egg-info',
+    '*.egg-info/*',
+    'setup.cfg',
+    '.hgtags',
+    '.hgsigs',
+    '.hgignore',
+    '.gitignore',
+    '.bzrignore',
+    '.gitattributes',
+    '.github/*',
+    '.travis.yml',
+    'Jenkinsfile',
+    '*.mo',
+    '.clang-format',
+    '.gitmodules',
+    'VERSION.txt',
+    '.editorconfig',
+    '*.yml',
+    '*.yaml',
+    'docs/*',
+    'docs/images/*',
+    'examples/*',
+    ]
+
+
+
+[tool.coverage]
+    [tool.coverage.run]
+        omit = [
+            '*_test.py',
+            '*_fixtures.py'
+        ]
+
+
+[tool.pylint]
+    [tools.pylint.master]
+    ignore-patterns = [
+        '__init__.py'
+    ]
+
+    [tools.pylint.format]
+    max-line-length = 120
```

Comment:
This was due out of habits since some tools have slightly different sensitivity for when they trigger. I will set everything to 120 to be more consistent and then see what happens.